jina/clients/python/request.py

```diff
@@ -82,11 +77,9 @@ def _generate(data: Union[Iterator[bytes], Iterator['jina_pb2.Document'], Iterat
             if mime_type:
                 d.mime_type = mime_type
 
-            d.doc_id = first_doc_id if not random_doc_id else random.randint(0, ctypes.c_uint(-1).value)
+            d.id = next(doc_counter)
```

Comment:
We will not support incrementing doc id assignment? I think it can be helpful for debugging purposes.

Reply:
okay I will refactor this part back using `SimpleCounter`